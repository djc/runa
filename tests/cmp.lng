def main(name: str, args: array[str]) -> int:
	
	print('bool')
	print(True)
	print(False)
	
	print('eq, int/bool')
	print(str(1 == 3))
	print(str(2 == 2))
	print(str(True == True))
	print(str(False == True))
	
	print('eq, str')
	print(str("a" == "b"))
	print(str("aa" == "b"))
	print(str("aa" == "bb"))
	print(str("a" == "a"))
	
	print('ne')
	print(str(1 != 4))
	print(str(False != False))
	print(str("a" != "b"))
	
	print('lt, int')
	print(str(1 < 4))
	print(str(2 < 2))
	print(str(5 < 4))
	
	print('lt, str')
	print(str('a' < 'aa'))
	print(str('a' < 'b'))
	print(str('a' < 'a'))
	print(str('b' < 'a'))
	
