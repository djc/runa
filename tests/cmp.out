bool
True
False
eq, int/bool
False
True
True
False
eq, str
False
False
False
True
ne
True
False
True
lt, int
True
False
False
lt, str
True
True
False
False
